javascript-closures/my-app/src/Counter.js

```diff
@@ -0,0 +1,15 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+export default function Counter() {
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    setCounter((prev) => prev + 1);
```

Comment:
You lost the call to getData. That's a problem since you are now not getting the data the component will need. So you need to add the call to `getData` back in here and put getData back in as a dependency.

Of course, in this particular case, it doesn't matter. This exercise, though, is intended to show you the pattern for how you do this.